ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionIdentitetStyrktGenom.java

```diff
@@ -0,0 +1,120 @@
+        return CertificateDataElement.builder()
+            .id(IDENTITET_STYRKT_GENOM_SVAR_ID_2)
+            .index(index)
+            .parent(IDENTITET_CATEGORY_ID)
+            .config(
+                CertificateDataConfigRadioMultipleCode.builder()
+                    .text(textProvider.get(IDENTITET_STYRKT_GENOM_TEXT_ID))
+                    .description(textProvider.get(IDENTITET_STYRKT_GENOM_DESCRIPTION_ID))
+                    .layout(Layout.COLUMNS)
+                    .list(
+                        Arrays.asList(
+                            RadioMultipleCode.builder()
+                                .id(IdKontrollKod.ID_KORT.getCode())
+                                .label(textProvider.get(IDENTITET_STYRKT_GENOM_ID_KORT_TEXT_ID))
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(IdKontrollKod.FORETAG_ELLER_TJANSTEKORT.getCode())
+                                .label(textProvider.get(IDENTITET_STYRKT_GENOM_FORETAG_ELLER_TJANSTEKORT_TEXT_ID))
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(IdKontrollKod.KORKORT.getCode())
+                                .label(textProvider.get(IDENTITET_STYRKT_GENOM_KORKORT_TEXT_ID))
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(IdKontrollKod.PERS_KANNEDOM.getCode())
+                                .label(textProvider.get(IDENTITET_STYRKT_GENOM_PERS_KANNEDOM_TEXT_ID))
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(IdKontrollKod.FORSAKRAN_KAP18.getCode())
+                                .label(textProvider.get(IDENTITET_STYRKT_GENOM_FORSAKRAN_KAP18_TEXT_ID))
+                                .build(),
+                            RadioMultipleCode.builder()
+                                .id(IdKontrollKod.PASS.getCode())
+                                .label(textProvider.get(IDENTITET_STYRKT_GENOM_PASS_TEXT_ID))
+                                .build()
+                        )
+                    )
+                    .build()
+            )
+            .value(
+                vardkontakt != null ? CertificateDataValueCode.builder()
+                    .id(vardkontakt.getIdkontroll())
+                    .code(vardkontakt.getIdkontroll())
+                    .build() : CertificateDataValueCode.builder().build()
+            )
+            .validation(
+                new CertificateDataValidation[]{
+                    CertificateDataValidationMandatory.builder()
+                        .questionId(IDENTITET_STYRKT_GENOM_SVAR_ID_2)
+                        .expression(
+                            multipleOrExpression(
+                                singleExpression(IdKontrollKod.ID_KORT.getCode()),
```

Comment:
If im not mistaken its redundant to use singleExpression here, maybe should skip it if its not needed?